src/libs/rover/partial_image.hpp

```diff
@@ -197,14 +202,14 @@ struct PartialImage
 #endif
     for(int i = 0; i < size; ++i)
     {
-      id_portal.Set(i, partials[i].m_pixel_id );
-      depth_portal.Set(i, partials[i].m_depth );
+      id_portal.Set(i, partials[i].m_pixel_id);
+      depth_portal.Set(i, partials[i].m_depth);
       const int starting_index = i * num_bins;
 
-      for(int ii = 0; ii < num_bins; ++ii)
+      for(int j = 0; j < num_bins; ++j)
       {
-        buffer_portal.Set(starting_index + ii, partials[i].m_bins[ii]);
-        intensity_portal.Set( starting_index + ii, partials[i].m_bins[ii] * background[ii]);
+        buffer_portal.Set(starting_index + j, partials[i].m_bins[j]);
+        intensity_portal.Set(starting_index + j, partials[i].m_bins[j]);
```

Comment:
I guess I'm confused; I thought we wanted to have the absorption times the background added to the intensity?